python/oneflow/test/modules/test_ctc_loss.py

```diff
@@ -311,6 +311,20 @@ def _test_ctc_loss_with_diff_device_input(test_case, reduction):
     return out
 
 
+
+@autotest(n=3, auto_backward=True, check_graph=False)
+def _test_ctc_loss_functional(test_case, reduction):
+    device_random = random_device()
+    log_probs = random_tensor(ndim=3, dim0=5, dim1=2, dim2=3).to(device_random)
+    targets = random_tensor(ndim=2, dim0=2, dim1=3, low=1, high=3, dtype=int).to(device_random)
+    input_lengths = torch.tensor([5, 5], dtype=torch.int32)
+    target_lengths = torch.tensor([3, 3], dtype=torch.int32)
+    out = torch.nn.functional.ctc_loss(log_probs, targets, input_lengths, target_lengths, reduction=reduction)
```

Comment:
```suggestion
    
    out = torch.nn.functional.ctc_loss(log_probs, targets, input_lengths, target_lengths, reduction=oneof('mean', 'none', nothing())
```

oneof 是 autotest 提供的工具函数之一